CMakePresets.json

```diff
@@ -7,7 +7,7 @@
       "generator": "Ninja",
       "binaryDir": "${sourceDir}/build/${presetName}",
       "cacheVariables": {
-        "CMAKE_CXX_STANDARD": "20"
+        "CMAKE_CXX_STANDARD": "23"
```

Comment:
what's the reason for this one?  I think we should leave it at 20

Reply:
to test import std;

Reply:
right, but given the limited compiler support and the hoop jumping required on the cmake side I don't see us putting import std on main yet -- that seems like it needs to wait until we have an official cmake that support import std?